libs/dh/admin/shared/src/dh-permissions-table.component.ts

```diff
@@ -32,31 +32,46 @@ import {
 } from '@energinet-datahub/watt/table';
 
 import { PermissionDetailsDto } from '@energinet-datahub/dh/shared/domain/graphql';
+import { WattEmptyStateComponent } from '@energinet-datahub/watt/empty-state';
+import { VaterFlexComponent } from '@energinet-datahub/watt/vater';
 
 @Component({
   selector: 'dh-permissions-table',
   standalone: true,
   template: `<ng-container *transloco="let t; read: 'admin.userManagement.permissionsTable'">
-    <watt-table
-      description="permissions"
-      [dataSource]="dataSource"
-      [columns]="columns"
-      [selectable]="permissions().length > 0"
-      [initialSelection]="initialSelection()"
-      sortBy="name"
-      sortDirection="asc"
-      (selectionChange)="selectionChanged.emit($event)"
-    >
-      <ng-container *wattTableCell="columns['name']; header: t('columns.name'); let element">
-        {{ element.name }}
-      </ng-container>
-
-      <ng-container
-        *wattTableCell="columns['description']; header: t('columns.description'); let element"
+    @if (permissions().length > 0) {
+      <watt-table
+        description="permissions"
+        [dataSource]="dataSource"
+        [columns]="columns"
+        [selectable]="permissions().length > 0"
```

Comment:
nitpick: this can be set to `true` since we're inside the `if`

```suggestion
        [selectable]="true"
```